docs/sources/auth/generic-oauth.md

```diff
@@ -80,6 +80,8 @@ Customize user login using `login_attribute_path` configuration option. Order of
 
 You can customize the attribute name used to extract the ID token from the returned OAuth token with the `id_token_attribute_name` option.
 
+Customize user's displayed name using `name_attribute_path` configuration option. Order of operations is the same as for the `login_attribute_path` option.
```

Comment:
Unless we'll squeeze this into 7.3, we should split out the docs changes to a separate PR that's merged together with 7.4-beta1 because our docs release pipeline publishes all docs changes immediately.

```suggestion
You can set the user's display name with JMESPath using the `name_attribute_path` configuration option. It operates the same way as the `login_attribute_path` option.

> **Note:** `name_attribute_path` is added in 7.4.
```

Reply:
Yes, forget about this, good catch!